packages/shared/src/utils/currency/currency.js

```diff
@@ -12,12 +12,25 @@ export const getFormattedText = (number, currency) => {
     return `${addComma(number, getDecimalPlaces(currency), isCryptocurrency(currency))} ${currency}`;
 };
 
-export const formatMoney = (currency_value, amount, exclude_currency, decimals = 0, minimumFractionDigits = 0) => {
-    let money = amount;
-    if (money) money = String(money).replace(/,/g, '');
+export const formatMoney = ({
+    currency_value,
+    amount,
+    exclude_currency,
+    decimals = 0,
+    minimumFractionDigits = 0,
+    should_format_crypto = true,
```

Comment:
If all cryptos, except IDK should be formatted, then why are we adding this additional check here? :thinking: I'm trying to wrap my head around the use case for this flag. If this is a flag to manipulate this function's behavior, should we set it as `false` on default? I think `false` would cover more use cases than `true`.  here.

See below question for more context.

Reply:
By default we want to format all currencies including cryptos so setting a default value of `true` for `should_format_crypto` makes sense. But in some cases (e.g. in Reports page) we want to keep current accuracy for cryptos so we need to pass a false value to `should_format_crypto` to change the normal functionality here.